src/openloco/localisation/string_ids.h

```diff
@@ -176,6 +178,9 @@ namespace openloco::string_ids
     constexpr string_id audio_device_none = 479;
     constexpr string_id stringptr = 480;
 
+    constexpr string_id spinner_up = 486;
```

Comment:
```suggestion
    constexpr string_id stepper_up = 486;
```

https://www.nngroup.com/articles/input-steppers/

Reply:
To me, the up/down ones are spinners, while the +/- ones are steppers. I will replace them with the latter later in this PR — I'd like to be full feature-parity first.